daemons/ipa-kdb/ipa_kdb_mspac.c

```diff
@@ -1798,8 +1798,12 @@ static krb5_error_code ipadb_verify_pac(krb5_context context,
         priv_key = krbtgt_key;
     }
 
-    kerr = krb5_pac_verify(context, old_pac, authtime,
-                            client_princ, srv_key, priv_key);
+    /* only pass with_realm TRUE when it is cross-realm ticket and S4U2Self
```

Comment:
Technically, cross-realm S4U as this also applies to cross-realm S4U2Proxy (aka RBCD).